src/api/svelte/TidySvelteApi.ts

```diff
@@ -1,9 +1,9 @@
 import { getContext, setContext, mount, unmount } from 'svelte';
 import { SvelteMap, SvelteSet } from 'svelte/reactivity';
-
 export const TidySvelteApi = {
   /**
    * Svelte resources which are directly tied to Tidy's svelte version / runtime.
+   * @deprecated potentially?
```

Comment:
Note: I added this just so I can include it as part of the conversation. I can remove this. Might need to think about if sharing components would even be possible considering the runtime sharing issue